### PR TITLE
22/Return item(s)

### LINKED ISSUE #22: Return item(s)
Return item(s) by scanning or entering its tag (a tag is a unique identifier, probably a barcode).

### PR DESCRIPTION
**Pull request by AdamVig**
This removes the need for any tests specific to the rental controller
because the endpoint service has full test coverage. This also adds
implementation for `rental.update()` and `rental.delete()`.

Closes #22.

### CODE REVIEW

**Comment by ghost** [comment]
Your pull request doesn't follow our guidelines. Please fix the following:

* Pull request title must be shorter than 51 characters [(?)](https://gitmagic.io/rules/#/pull-request/subject-must-be-shorter-than)

[Click here](https://app.gitmagic.io/AdamVig/stockpile-api/83/75de7ac9-4433-46e3-83ff-643536c63c9c/a8f20740-8d50-4584-a99c-ddc881576ffb) for details.

Thank you! :pray:
<sub>
This comment was made by [GitMagic](https://gitmagic.io/) – Magically enforcing your contribution guidelines.
</sub>

**Comment by ghost** [comment]
Thank you, the title and description now looks good! :bowtie:

File: controllers/rental.js [hunk 1]
```diff
@@ -1,31 +1,10 @@
...
-}
-
-module.exports.delete = (req, res, next) => {
-  res.send({})
-}
+module.exports.getAll = endpoint.getAll(table)
+module.exports.get = endpoint.get(table, key)
+module.exports.create = endpoint.create(table, 'rental created')
+module.exports.update = endpoint.update(table, key)
+module.exports.delete = endpoint.delete(table, key, 'rental deleted')
```
Comment (emroussel) [inline controllers/rental.js:10]: So how will the delete rental work? Will the front end have to send a `DELETE` request for every item the user wants to return?

**Review by emroussel — CHANGES_REQUESTED** [review]

File: controllers/rental.js (hunk 1 above)
Comment (AdamVig) [reply controllers/rental.js:10]: Yes!

**Review by emroussel — APPROVED** [review]

**Pull request merged**